driver-adapters-wasm/pg-cfpages-basic/run.sh

```diff
@@ -2,9 +2,10 @@
 
 set -eu
 
+echo "Temporary disabled, see https://github.com/cloudflare/workers-sdk/pull/2541"
```

Comment:
```suggestion
echo "Temporary disabled, because wrangler does not support "--node-compat" flag for pages commands yet. See https://github.com/cloudflare/workers-sdk/pull/2541"
```